<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionMotsDePasse</class>
 <widget class="QDialog" name="GestionMotsDePasse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>341</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>461</width>
    <height>341</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>461</width>
    <height>341</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestionnaire de Mots de passe - Gestion des mots de passe</string>
  </property>
  <widget class="QLabel" name="titrePage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Gestion des Mots de passe</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>441</width>
     <height>291</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page1">
    <widget class="QPushButton" name="supprimer">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>230</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer le mot de passe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="creer">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Créer un mot de passe</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listeMotsDePasse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="modifier">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>170</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier le mot de passe</string>
     </property>
    </widget>
    <widget class="QLabel" name="titreSelection">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Sélection</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="gererGroupes">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Gérer les affectations
aux groupes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="actualiser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Actualiser</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page2">
    <widget class="QLabel" name="titreCreerModifier">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Créer|Modifier un mot de passe</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="identifiant">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="motdepasse">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelIdentifiant">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Identifiant :</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMotDePasse">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mot de passe :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="validerMotDePasse">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
    <widget class="QPushButton" name="annulerMotDePasse">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelServeur">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Serveur :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="serveur">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelNom">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page3">
    <widget class="QLabel" name="titreGererGroupes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Gérer les groupes</string>
     </property>
    </widget>
    <widget class="QListWidget" name="groupes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>181</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="groupesMotDePasse">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>181</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelGroupes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Liste des groupes</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelGroupesMotDePasse">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Groupes ayant accès au mot de passe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouterGroupe">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="enleverGroupe">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>180</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="validerGroupes">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>260</y>
       <width>86</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
    <widget class="QPushButton" name="annulerGroupes">
     <property name="geometry">
      <rect>
       <x>345</x>
       <y>260</y>
       <width>86</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMDP">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Mot de passe</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>creer</tabstop>
  <tabstop>listeMotsDePasse</tabstop>
  <tabstop>actualiser</tabstop>
  <tabstop>gererGroupes</tabstop>
  <tabstop>modifier</tabstop>
  <tabstop>supprimer</tabstop>
  <tabstop>nom</tabstop>
  <tabstop>identifiant</tabstop>
  <tabstop>motdepasse</tabstop>
  <tabstop>serveur</tabstop>
  <tabstop>validerMotDePasse</tabstop>
  <tabstop>annulerMotDePasse</tabstop>
  <tabstop>groupes</tabstop>
  <tabstop>groupesMotDePasse</tabstop>
  <tabstop>ajouterGroupe</tabstop>
  <tabstop>enleverGroupe</tabstop>
  <tabstop>validerGroupes</tabstop>
  <tabstop>annulerGroupes</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
